GitCommands/UserRepositoryHistory/Repository.cs

```diff
@@ -10,7 +10,7 @@ public class Repository
         public enum RepositoryAnchor
         {
             MostRecent,
```

Comment:
So would you say these are "pinned"?

Reply:
+100
I was about to ask for renaming to "pinned" also...

Reply:
Another massive change...
Rebased too.